src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php

```diff
@@ -78,11 +79,11 @@
     $PHP .= ' -qrr';
 }
 
-$defaultEnvs = array(
+$defaultEnvs = [
```

Comment:
please revert all changes from fabbot, they're not applicable to this particular bridge